app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -64,15 +67,26 @@ class AdministratorControlsViewModel @Inject constructor(
   private fun processAdministratorControlsList(
     deviceSettings: DeviceSettings
   ): List<AdministratorControlsItemViewModel> {
-    val itemViewModelList: MutableList<AdministratorControlsItemViewModel> = mutableListOf(
-      AdministratorControlsGeneralViewModel()
-    )
-    itemViewModelList.add(
-      AdministratorControlsProfileViewModel(
-        routeToProfileListListener,
-        loadProfileListListener
+
+    val itemViewModelList: MutableList<AdministratorControlsItemViewModel>
+
+    if (enableEditAccountsOptionsUi.value) {
+      itemViewModelList = mutableListOf(
+        AdministratorControlsGeneralViewModel(),
+        AdministratorControlsProfileViewModel(
+          routeToProfileListListener,
+          loadProfileListListener
+        )
       )
```

Comment:
```suggestion
    val itemViewModelList = mutableListOf<AdministratorControlsItemViewModel>()

    if (enableEditAccountsOptionsUi.value) {
      itemViewModelList += AdministratorControlsGeneralViewModel()
```

More Kotlin-idiomatic.